File: Camp codes/Diffent tasks 1/Факториал (фул).cpp
#include <iostream>

using namespace std;

int main()
{
    int cnt,N,dilnik;
    cin >> N;
    dilnik=5;
    cnt=0;

    while (dilnik <= N)
    {
        cnt=cnt + (N/dilnik);
        dilnik=dilnik*5;
    }

    cout << cnt;

    return 0;

}

File: Camp codes/Diffent tasks 1/Факториал (фул).pas
var N,dilnik,cnt:longint;

begin
  cnt:=0;
  readln(N);
  dilnik:=5;
  while dilnik<=n do
  begin
     cnt:=cnt + (n div dilnik);
     dilnik:=dilnik*5;
  end;
  
  writeln(cnt);
end.